orchestra/utils/task_lifecycle.py

```diff
@@ -1194,6 +1194,7 @@ def create_subsequent_tasks(project):
         project (orchestra.models.Project):
             The modified project object.
     """
+    end_project = False
```

Comment:
style: i'd define this next to line 1207, so it's defined in a similar spot as machine_tasks_to_schedule, which also gets filled in during the loop over all steps

Reply:
done